aws/scenarios/vm/os/macos.go

```diff
@@ -3,46 +3,46 @@ package os
 import (
 	"github.com/DataDog/test-infra-definitions/aws"
 	"github.com/DataDog/test-infra-definitions/aws/ec2/ec2"
+	"github.com/DataDog/test-infra-definitions/common/os"
 )
 
 type macOS struct {
+	*os.MacOS
```

Comment:
TIL: field names are not mandatory in struct, by default go uses field type as a name

Reply:
`*os.MacOS` is an [embedded struct](https://eli.thegreenplace.net/2020/embedding-in-go-part-1-structs-in-structs/). The usage of an embedded struct and a regular field is not the same. 
Embedded struct should be used only when all the methods from the embedded struct (`*os.MacOS`) should be accessible from parent type (`macOS`).